src/configure-tests/feature-tests/blk_mq_make_request.c

```diff
@@ -0,0 +1,17 @@
+// SPDX-License-Identifier: GPL-2.0-only
+
+/*
+ * Copyright (C) 2021 Elastio Software Inc.
+ */
+
+// 5.8 == kernel_version
+
+#include "includes.h"
+MODULE_LICENSE("GPL");
```

Comment:
This feature test doesn't compile without  `#include <linux/blk-mq.h>` neither on the kernel 5.8 nor on the kernel 4.18-0.305. Please find `HAVE_BLK_MQ_MAKE_REQUEST - not present` in the build logs in the PR for [CentOS 8](https://github.com/elastio/elastio-snap/pull/85/checks?check_run_id=3383931593#step:9:44) and [Fedora 31](https://github.com/elastio/elastio-snap/pull/85/checks?check_run_id=3383931674#step:9:44). As result, the tests are hung and [build doesn't pass](https://github.com/elastio/elastio-snap/pull/85/checks?check_run_id=3383931593#step:11:24).
```suggestion
#include "includes.h"
#include <linux/blk-mq.h>
MODULE_LICENSE("GPL");
```